app/grandchallenge/cases/tasks.py

```diff
@@ -156,12 +152,23 @@ def build_images(  # noqa:C901
                 DisplaySet._meta.model_name,
             ]:
                 # users can delete archive items and display sets before this task runs
+                logger.info(
+                    f"Nothing to do here: {linked_model_name} no longer exists."
+                )
+                upload_session.update_status(
+                    status=RawImageUploadSession.CANCELLED
```

Comment:
maybe add an error message here for the user? Something along the lines of "Linked object no longer exists."

Reply:
We could, but then the user get's informed about it. Do we really want that?

Reply:
Well, it's not secret, right? They deleted the object, so they know already. 
I agree that it's not strictly necessary, but it might help them understand why certain uploads were cancelled if they later look at their overview.

Reply:
I am in doubt since generally they aren't even aware that an upload is being processed in the backend.  I doubt, but you seem certain so let's roll with a notification!

Reply:
Done!